Fastfile

```diff
@@ -117,30 +117,14 @@ def _build_ipa
 end
 
 def _build_with_gym
-  install_provisioning_profiles
   _update_team_id_if_necessary
-  export_method = _get_export_method
-  xcconfig_filename = Dir.pwd + '/TAB.release.xcconfig'
-  create_xcconfig(filename: xcconfig_filename)
-  gym(export_method: export_method, xcconfig: xcconfig_filename)
-end
-
-def _get_export_method
-  if ENV['GYM_EXPORT_OPTIONS'].nil?
-    _fallback_to_enterprise_export_method
-  else
-    get_info_plist_value(path: ENV['GYM_EXPORT_OPTIONS'], key: 'method')
-  end
-end
-
-def _fallback_to_enterprise_export_method
-  UI.message('Falling back to enterprise `export_method` since `GYM_EXPORT_OPTIONS` is not defined')
-  'enterprise'
+  update_project_provisioning
+  gym
 end
 
 def _update_team_id_if_necessary
   project_path = ENV['FL_PROJECT_SIGNING_PROJECT_PATH']
-  team_id = _get_team_id
+  team_id = ENV['FL_PROJECT_TEAM_ID']
```

Comment:
If the team id env variable is empty by accident is there a backup way we could auto-detect it?

Reply:
The way we were doing this before was to investigate the export options plist to see if it was there, this PR aims to remove the need for a developer to supply this